internal/querycoord/task.go

```diff
@@ -2491,7 +2491,7 @@ func assignInternalTask(ctx context.Context,
 				baseTask.setParentTask(parentTask)
 				loadSegmentTask := &loadSegmentTask{
 					baseTask:            baseTask,
-					LoadSegmentsRequest: batchReq,
+					LoadSegmentsRequest: req,
```

Comment:
I don't think this shall be `req` instead of `batchReq`. This part is actually packing the last batch requests.

Reply:
3 possible cases:
- the threshold of etcd is less than MaxSendSizeToEtcd
- proto.Size(batch) + proto.Size(req) is less than the batch merged req
- single req is large enough to break the threshold